docs/vocabulary.md

```diff
@@ -3,6 +3,7 @@
 - [Introduction](#introduction)
 - [Terminology Used by CI/CD Tools and Technologies](#terminology-used-by-cicd-tools-and-technologies)
   - [ArgoCD](#argo-cd)
+  - [Codefresh](#codefresh)
```

Comment:
Sorry for nitpicking but I'm wondering if you could move Codefresh after CircleCI to continue listing technologies/projects alphabetically?

Reply:
Not nitpicking, moved it further down